cfme/physical/provider/redfish.py

```diff
@@ -0,0 +1,69 @@
+import attr
+
+from widgetastic_patternfly import BootstrapSelect, Input
+from wrapanapi.redfish import RedfishSystem
+
+from cfme.common.provider import DefaultEndpoint, DefaultEndpointForm
+from . import PhysicalProvider
+
+
+class RedfishEndpoint(DefaultEndpoint):
+    api_port = 8000
+    security_protocol = 'Non-SSL'
```

Comment:
I think setting defaults here makes no sense because everything will crash if you omit a key from cfme_data, but I may be wrong.

Reply:
The defaults work OK if they are omitted from `cfme_data.yaml`. However, the test _does_ crash later in `RedfishProvider.mgmt` if I don't declare these fields. I suppose that's a particularity of the `PhysicalProvider`.